packaging/nmstate.py3.spec

```diff
@@ -24,10 +24,12 @@ provider support on the southbound.
 %package -n python3-%{libname}
 Summary:        nmstate Python 3 API library
 Requires:       python3-dbus
-Requires:       NetworkManager-libnm >= 1:1.12
+Requires:       NetworkManager-libnm >= 1:1.20
 # Use Recommends for NetworkManager because only access to NM DBus is required,
 # but NM could be running on a different host
 Recommends:     NetworkManager
```

Comment:
Should we request NM 1.20 (like we do with libnm)?

Reply:
NM has a versioned dependency on libnm, therefore libnm >= 1.20 implies that NM also needs to be updated (if it is installed).